src/components/secondPassphraseInput/secondPassphraseInput.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Input from 'react-toolbox/lib/input';
+import { isValidPassphrase } from '../../utils/passphrase';
+
+class SecondPassphraseInput extends React.Component {
+  componentDidMount() {
+    if (this.props.hasSecondPassphrase) {
+      this.props.onError(undefined, '');
+    }
+  }
+
+  handleChange(value) {
+    this.props.onChange(value);
+    const error = this.validate(value);
+    if (error) {
+      this.props.onError(error, value);
+    }
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  validate(value) {
```

Comment:
This `eslint` check is IMO stupid. My reason was that the function logically belongs to the component. There is a very similar `validateInput` function in send component [0], which by chance needs to use `this` so `eslint` is not complaining. But both functions are from the semantic point of view the same (validators) and thus both should IMO be also syntactically the same. 

[0] https://github.com/LiskHQ/lisk-nano/blob/development/src/components/send/send.js#L51-L62